elasticache/elasticache.tf

```diff
@@ -80,3 +80,21 @@ resource "aws_elasticache_replication_group" "cluster_mode" {
     ]
   }
 }
+
+resource "aws_elasticache_parameter_group" "main" {
+  count = var.maxmemory_policy == null ? 0 : 1
+
+  name = local.cluster_name
+  # Family need to be specified so that we can copy keys n values from the existing parameter group
```

Comment:
can we document a few examples of which part we return, e.g. when "parameter_group_name" is
* `default.redis6.x` => `redis6.x`
* `default.redis6.x.cluster.on` => `redis6.x`

Reply:
Good ideas! Have added it into the comments :)